catalog/rocks/entry_catalog.go

```diff
@@ -134,6 +134,15 @@ func (e *EntryCatalog) CommitExistingMetaRange(ctx context.Context, repositoryID
 	return e.store.CommitExistingMetaRange(ctx, repositoryID, branchID, metaRangeID, committer, message, metadata)
 }
 
+func (e *EntryCatalog) AddCommitNoLock(ctx context.Context, repositoryID graveler.RepositoryID, commit graveler.Commit) (graveler.CommitID, error) {
```

Comment:
Please document (any function that contains the words "NoLock").

Reply:
True, I documented just the interface 🤔